mne/coreg.py

```diff
@@ -297,21 +297,7 @@ def _decimate_points(pts, res=10):
 
 
 def _trans_from_params(param_info, params):
-    """Convert transformation parameters into a transformation matrix.
-
-    Parameters
-    ----------
-    param_info : tuple,  len = 3
-        Tuple describing the parameters in x (do_translate, do_rotate,
-        do_scale).
-    params : tuple
-        The transformation parameters.
-
-    Returns
-    -------
-    trans : array, shape = (4, 4)
-        Transformation matrix.
-    """
+    """Convert transformation parameters into a transformation matrix."""
```

Comment:
Private docstrings almost never stay up to date -- we don't review them carefully (onerous) and we also don't have any automated checking for them. Over time I've been removing these in PRs, and most of the time there is something wrong with them. In this case, the boolean parameter order was wrong for the first argument. We could try to make it so that all private functions are documented 100% correctly, but in the end this might slow us down more than help us out 🤷

Reply:
oh, no, I would not suggest that! :)
I just remember that on a few occasions I had to traverse large parts of the code to figure out the inputs to a given private function. So I'm ok with reducing these docstrings, especially if they're out of date as long as we don't remove something very non-obvious that was already present.

Reply:
What we really need are type annotations but this is a longer discussion :)

Reply:
Indeed when I remove these I usually try to improve the code commenting if the docstring has something non-obvious.

@hoechenberger feel free to open a separate issue about type annotations if we don't have one already